tests/config/environment.yml

```diff
@@ -2,7 +2,6 @@ name: pytest-env-CI
 channels:
   - conda-forge
   - bioconda
-  - defaults
```

Comment:
We don't have more than 200 people in our organization, but most folks are moving away from using the `defaults` channel due to Anaconda updating their ToS: https://www.anaconda.com/blog/is-conda-free

Reply:
Thanks for picking up on this & making the quick fix!